pyproject.toml

```diff
@@ -35,6 +35,7 @@ dependencies = [
     "opencv-python-headless",
     "quads>=1.1.0",
     "numpy<2",
+    "scikit-image==0.24.0"
```

Comment:
move to optional dependencies + add corresponding documentation to README

Reply:
Done.